ohapi/command_line.py

```diff
@@ -237,9 +240,19 @@ def upload_metadata(directory, create_csv='', review='',
     set_log_level(debug, verbose)
 
     max_bytes = parse_size(max_size)
-
-    if create_csv:
+    if create_csv and review:
+        raise ValueError("Either create_csv must be true or review must be " +
+                         "true but not both")
+    if review:
+        if review_metadata_csv(directory, review, max_bytes=max_bytes):
+            print('csv is in the desired format')
+        else:
+            print('csv is not in the desired format')
```

Comment:
It would be much more helpful for users to have an indicator of the type of error they've encountered. For example…
* "Error, row 4: MD5 sum field is provided but appears to be incorrectly formatted"
* "Error: 'description' column not provided"
* "Warning, row 4: Number of columns (7) does not match header row length (5)

I think I'd suggest accomplishing this by raising `error` and `warning` statements in functions this calls, then catching them and returning them here.

Sorry to be nitpicky! But I suspect a data validation command is going to be really frustrating for users if it doesn't provide hints on what is wrong in the data they're trying to validate. :)

Reply:
@madprime , If error is given on which project member id and filename is caused rather than on which rows, it is caused other part of error remains same. Will this be fine to you ?